src/bitboxbase/bitboxbase_screensaver.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef _BITBOXBASE_SCREENSAVER_H
+#define _BITBOXBASE_SCREENSAVER_H
+
+#include <stdbool.h>
+
+void bitboxbase_screensaver_init(void);
```

Comment:
docstrings missing for the three functions.

Reply:
ack